MekHQ/src/mekhq/gui/panes/UnitMarketPane.java

```diff
@@ -469,6 +482,50 @@ private void finalizeEntityAcquisition(final List<UnitMarketOffer> offers,
         getMarketModel().setData(getCampaign().getUnitMarket().getOffers());
     }
 
+    /**
+     * Returns the quality of a unit based on the given market type.
+     *
+     * @param market the type of market
+     * @return the quality of the unit
+     */
+    private int getQuality(UnitMarketType market) {

```

Comment:
Do we want to have this "business logic" code in this UI class, or maybe move it to the UnitMarketType class instead?